less/_defaults/_spacing-containers.less

```diff
@@ -19,7 +19,12 @@
 @menu-padding-hoz: 0;
 
 @menu-header-padding-ver: 2rem;
-@menu-header-padding-hoz: .5rem;
+@menu-header-padding-hoz: 1rem;
+
+@menu-item-container-padding-hoz: 1rem;
```

Comment:
Is `@menu-item-container-padding-hoz` needed @StuartNicholls?

Reply:
> Is `@menu-item-container-padding-hoz` needed @StuartNicholls?

Nope, added that in when thinking I could fix the menu items. Removing it for now